tests/integration/components/disputeManagement/notificationOfFraud.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test, expect } from '@playwright/test';
+import { goToStory } from '../../../utils/utils';
+
+const STORY_ID = 'mocked-dispute-management--notification-of-fraud';
+
+test.describe('Notification of fraud', () => {
+    test.beforeEach(async ({ page }) => {
+        await goToStory(page, { id: STORY_ID });
+    });
+
+    test('should render a contact support alert when dispute is not defensable trough the component', async ({ page }) => {
+        await expect(page.locator('.adyen-pe-tag--default', { hasText: 'Notification of fraud' })).toBeVisible();
+        await expect(page.getByRole('alert')).toBeVisible();
+
+        const icon = page.locator('.adyen-pe-alert__icon');
+        await icon.waitFor();
```

Comment:
Ditto: previous comment
```suggestion
        await icon.waitFor({ state: 'visible' });
```

Reply:
✅